stdlib/math.jou

```diff
@@ -1,40 +1,112 @@
+# Math lib
+#TODO: add test
+
+# Other functions
+# Returns the absolute value of x.
 declare abs(x: int) -> int
 declare llabs(x: long) -> long
 declare fabs(x: double) -> double
+# Returns x*y+z.
+declare fma(x: double, y: double, z: double) -> double
 
+# Rounding and remainder functions
+# Rounds x upward, returning the smallest integral value that is not less than x.
 declare ceil(x: double) -> double
+# Rounds x downward, returning the largest integral value that is not greater than x.
 declare floor(x: double) -> double
+# Returns the floating-point remainder of numer/denom (rounded towards zero):
+# fmod = numer - tquot * denom
```

Comment:
Instead of adding the `fmod` function, we should make the `%` operator work  with floats. I'm not sure if it currently works with floats.

Reply:
We can remove it when "%" operator work with floats.